exercises/pascals-triangle/pascals_triangle_test.cpp

```diff
@@ -0,0 +1,103 @@
+#include "pascals_triangle.h"
+#define BOOST_TEST_MAIN
+#include <boost/test/unit_test.hpp>
+
+BOOST_AUTO_TEST_CASE(zero_rows)
+{
+    const std::vector<std::vector<int>> expected;
+
+    const std::vector<std::vector<int>> actual{pascals_triangle::gen(0)};
+
+    BOOST_TEST(expected == actual);
+}
+
+#if defined(EXERCISM_RUN_ALL_TESTS)
+
+BOOST_AUTO_TEST_CASE(single_row)
+{
+    const std::vector<std::vector<int>> expected{{1}};
+
+    const std::vector<std::vector<int>> actual{pascals_triangle::gen(1)};
+
+    BOOST_TEST(expected == actual);
+}
+
+BOOST_AUTO_TEST_CASE(two_rows)
+{
+    const std::vector<std::vector<int>> expected{{1}, {1,1}};
+
+    const std::vector<std::vector<int>> actual{pascals_triangle::gen(2)};
+
+    BOOST_TEST(expected == actual);
+}
+
+BOOST_AUTO_TEST_CASE(three_rows)
+{
+    const std::vector<std::vector<int>> expected{{1}, {1,1}, {1,2,1}};
+
+    const std::vector<std::vector<int>> actual{pascals_triangle::gen(3)};
+
+    BOOST_TEST(expected == actual);
+}
+
+BOOST_AUTO_TEST_CASE(four_rows)
+{
+    const std::vector<std::vector<int>> expected{{1},{1,1},{1,2,1},{1,3,3,1}};
+
+    const std::vector<std::vector<int>> actual{pascals_triangle::gen(4)};
+
+    BOOST_TEST(expected == actual);
+}
+
+BOOST_AUTO_TEST_CASE(five_rows)
+{
+    const std::vector<std::vector<int>> expected{
+      {1},
+      {1,1},
+      {1,2,1},
+      {1,3,3,1}, 
```

Comment:
Please match the formatting across the different arrays here and in the other places in the test suite.

Reply:
Its okay if I use this formatting right? (with each row on a new line?) I have made it consistent across the different tests.

Reply:
I actually meant the spacing inside the array, as some of your arrays have spaces after commas and some of them do not. (This is partially my fault for not setting up an auto-formatter).

I'll just edit it in the web interface myself, it's a tiny change.